NEWS.md

```diff
@@ -38,6 +38,24 @@
 * Update C++ unit testing library (catch) to v1.3.3 (fix gcc >=5 compilation errors)
 * Syntax highlighting of raw, wide, unicode string literals (e.g R"hello")
 
+### Profiler
```

Comment:
Let's lean this down a bit, maybe 5 or 5 bullets summarizing the gist of our profiling features.